modules/andes-core/client/src/main/java/org/wso2/andes/client/BasicMessageConsumer.java

```diff
@@ -630,11 +630,7 @@ public void close(boolean sendClose) throws JMSException
                         // If the session is open or we are in the process
                         // of closing the session then send a cance
                         // no point otherwise as the connection will be gone
-                        if (!_session.isClosed() || _session.isClosing())
-                        {
-                            if(null != _session._dispatcher) {
-                                _session._dispatcher.rejectPending(this);
-                            }
+                        if (!_session.isClosed() || _session.isClosing()) {
```

Comment:
Any reason for using '_' value with variables ?

Reply:
This is variable is coming from Qpid code which is  a external dependency.